src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -1246,10 +1259,54 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 		return false;
 	}
 
+	// Track and manage gpio command writes
+	bool write_gpio_command = false;
+
+	if (_gpio_ctl_en) {
+		if (_gpio_ctl_high != prev_gpio_ctl_high) {
+			gpio_write_counter = 0;
+		}
+
+		if (gpio_write_counter < 10) {
+			write_gpio_command = true;
+			gpio_write_counter++;
+		}
+
+		prev_gpio_ctl_high = _gpio_ctl_high;
+
+		if (write_gpio_command) {
+			Command gpio_cmd;
+			const int ESC_PACKET_TYPE_GPIO_CMD = 15;
+			uint8_t data[5];
+
+			int esc_id = 0; // In future un-hardcode
+			int val = 0;
+
+			if (_gpio_ctl_high) {
+				val = 1;
+			}
+
+			data[0] = esc_id; // esc id
+			data[1] = 80; // 01010000 : pin F0
+			data[2] = 0; // 0: output, 1: input
+			data[3] = val; //cmd LSB
+			data[4] = 0; // cmd MSB
+
+			// type, data, size
+			gpio_cmd.len = qc_esc_create_packet(ESC_PACKET_TYPE_GPIO_CMD, (uint8_t *) & (data[0]), 5, gpio_cmd.buf,
+							    sizeof(gpio_cmd.buf));
+
+			if (_uart_port.write(gpio_cmd.buf, gpio_cmd.len) != gpio_cmd.len) {
+				PX4_ERR("VOXL_ESC: Failed to send gpio packet");
```

Comment:
PX4_ERR (and PX4_WARN/PX4_INFO) include the module in the output.

```suggestion
				PX4_ERR("Failed to send gpio packet");
```

Reply:
@dagar I removed the extraneous VOXL_ESC module tags.